fluentd/configs.d/input/old-default-syslog.conf

```diff
@@ -23,7 +23,7 @@
 <source>
   @type tail
   @label @INGRESS
-  path /var/log/messages*
+  path /var/log/messages
```

Comment:
This will make it so we are only reading in the current syslog messages.  I'd recommend that we instead add an `exclude_path`

```
exclude_path ["/path/to/*.gz", "/path/to/*.zip"]
```